### DISCUSSION TITLE
lista de problemas com responsividade

### DISCUSSION THREAD

**Discussion by JjJjJose** [PewDizinho/PI1#21] [Ideas]
@Trizottetess e o brunao 

aqui vai uma lista pra guiar os manos na responsividade e nao terem que ficar perdendo tempo procurando, qualquer bug despercebido e so avisar :^)

1. Index.html[⭐ ]
-a partir de 750px a barra de navegação com a logo, home, area do cliente, começa a ficar muito junto (vale para todas as index) [✔️]
-com 1285px "Categorias" sai da box[✔️]
-a partir de 1100px pra baixo os itens das categorias começam a sair da box (colocar um collapse trava muito isso)[✔️]
-o balao de perfil ta em baixo wtfkkkkkkkkkkkkkkkkkkk [✔️] 
-com 1285px o 69 do vendas realizadas este mes fica um pouco para fora [✔️] 
-com 935px os textos escritos nos cards dos vendedores do mês vao magicamente para baixo [✔️]
-adicionar um limite de caracteres nos cards dos vendedores do mês [✔️] 

1.2. ideias que ficariam chave 😎 para organizar essa parte[⭐ ]
-colocar um "collapse" ou "dropdown" no categorias (bootstrap) [✔️]
-colocar em sequencia 1.categorias 2.vendedores do mes 3. balao de perfil [✔️]

2. Pasta pages -> about -> index.html  [⭐ ] 
-as fotos viram literalmente um ovo quanto mais diminui a guia [✔️]

3. Pasta pages -> clientplace -> index.html [⭐ ]
-o balao guest vai diminuindo conforme o navegador diminui, da para deixar um tamanho fixo se pa [✔️]

4. Pasta pages -> clientplace -> client -> index.html[⭐ ]
-da para deixar em sequencia 1.perfil 2. meus anuncios 3. sobre [✔️] 

5.Pasta pages -> clientplace -> client -> myaccount -> index.html[⭐ ]
-da para deixar os botoes de twitter, youtube e github embaixo da foto de perfil [✔️] 
-ir afunilando as box's uma em baixo da outra [✔️]

6.Pasta pages -> clientplace -> vendedor-> index.html
-1285px os elementos do painel de controle saem para baixo
-970px um dos textos começa a sair
-de 750px pra baixo o texto painel de controle vai começar a sair 
-a barra de pesquisa começa a sair de lugar tambem

6.1 ideias fodas 
-da pra reaproveitar as ideias de 1.2 

7.Pasta pages -> clientplace -> vendedor -> myaccount -> index.html[⭐ ]
-copiar o codigo inteiro de 5.[✔️]

8.Pasta pages -> contact -> index.html [⭐ ]
-deixar os textos centralizados em cima das caixas de texto com um tamanho fixo [✔️]
-o botao de enviar formulario diminui com a pagina [✔️] 
-os textos magicamente vao para a direita [✔️] 
-acho melhor definirmos um tamanho fixo para esses elementos[✔️]

9.Pasta pages -> login -> registro -> index.html [⭐ ]
-as boxs vao diminuindo com a guia[✔️]
-os textos escapam pra fora da box[✔️]
-da para deixar um embaixo do outro [✔️]

10.Pasta pages -> login -> cliente -> index.html  [⭐ ]
-tem tudo de errado essa porra nem sei o que falar pra ser sincero [✔️]

**Comment by JjJjJose** [comment]
não esqueça de marcar com um [✔️]

**Comment by PewDizinho** [comment]
Movendo isso pra "Discussions" (https://github.com/PewDizinho/PI1/discussions/21), eu fico recebendo notificação por ter um issue sobre o projeto aberto, e faz mais sentido ficar lá

**Comment by PewDizinho** [comment]
Algumas coisas que eu mexi aqui:

![image](https://github.com/PewDizinho/PI1/assets/55335712/45ae6fec-1310-40b9-b981-f0c9f148b65d)

Algumas mudanças na aba esquerda de categoria


RESPONSIVDADE:

A partir de Width: 980px o menu inicial muda para isso aqui:
![image](https://github.com/PewDizinho/PI1/assets/55335712/de4e9bee-753a-4a77-82cf-ed4a3132a22a)

Removi a navBar e adicionei esse botão de navegação simples de exemplo, mexer nisso até ficar bacana
![image](https://github.com/PewDizinho/PI1/assets/55335712/13cf1348-b3ea-4c38-859b-e0d3d89164de)


O minimo que o site aguenta é uma width de 225px, mas duvido que exista um telefone desse tamanho
![image](https://github.com/PewDizinho/PI1/assets/55335712/77025606-57e6-44ee-b043-f0621b05bc87)
[157px]
